internal/conventionalcommits/conventional_commits_test.go

```diff
@@ -35,7 +35,7 @@ func TestParseCommits(t *testing.T) {
 		wantErrPhrase string
 	}{
 		{
-			name:    "simple feat",
+			name:    "simple commit with no library association",
```

Comment:
Could you change the space to underscore? It's easy to locate the test case when test failed.

```suggestion
			name:    "simple_commit_with_no_library_association",
```

Reply:
sounds good, will do